### PR TITLE
Add 'never' to secondary_info: last-triggered

### PR DESCRIPTION
**Pull request by Mariusthvdb**
right now we don't have a value if the automation or script hasn't been triggered yet, and last-triggered is set for secondary_info. This seeks to add showing 'never' in that case.

Schermafbeelding 2020-01-07 om 10 32 47

### CODE REVIEW

File: src/panels/lovelace/components/hui-generic-entity-row.ts [hunk 1]
```diff
@@ -113,7 +113,7 @@ class HuiGenericEntityRow extends LitElement {
                     .datetime=${stateObj.attributes.last_triggered}
                   ></ha-relative-time>
                 `
-              : ""}
+              : "never"}
```
Comment (bramkragten) [inline src/panels/lovelace/components/hui-generic-entity-row.ts]: Please use localization

Comment (bramkragten) [reply src/panels/lovelace/components/hui-generic-entity-row.ts]: You now show never when there is anything else than `last-triggered` in the config for `secondary-info`

Comment (Mariusthvdb) [reply src/panels/lovelace/components/hui-generic-entity-row.ts]: feared as much, that's not what I wanted of course... sorry.
Am afraid this is above my current skill set... could you give me a hint please? Do we need a test on last_triggered to exist, ie !== null ?

like:

              : this.config.secondary_info === "last-triggered"
                :stateObj.attributes.last_triggered !== null
                ? html`
                    <ha-relative-time
                      .hass=${this.hass}
                      .datetime=${stateObj.attributes.last_triggered}
                     ></ha-relative-time>
                  `
                : "never"
same goes for the localization. I need to have an example so I can check. Please let me know if I can change this in the PR like this. thanks.

Comment (iantrich) [reply src/panels/lovelace/components/hui-generic-entity-row.ts]:
```suggestion
              : this.config.secondary_info === "last-triggered"
              ? stateObj.attributes.last_triggered
              ? html`
                  <ha-relative-time
                    .hass=${this.hass}
                    .datetime=${stateObj.attributes.last_triggered}
                  ></ha-relative-time>
                `
              : this.hass.localize("ui.panel.lovelace.entities.never_last_triggered")
              : ""
```
You'll need to add the translation field to [src/translationsen.json](https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/translations/en.json) and probably come up with a better key value name.

Comment (Mariusthvdb) [reply src/panels/lovelace/components/hui-generic-entity-row.ts]: thanks Ian!
id like to keep it short and simple, so prefer "never". wouldn't that suffice? anyone adding last-triggered for secondary_info: would know what that means.

**Comment by Mariusthvdb** [comment]
Alert... don't know what went wrong, but the last edit I tried to revert to Ians suggestion, and now the complete file seems to have been deleted...

my suggestion would be to change "never_last_triggered" to "never_triggered".

could Ian or Bram please reset the file to a point I can make that edit?

**Comment by bramkragten** [comment]
Ian's code was supposed to replace the old code, not just the "never" string. I'll close this, please create a new PR.
